modalContent.html

```diff
@@ -0,0 +1,6 @@
+<div>
+  Content
+  <div class="loader" ng-show="!media.loaded">
+    Loading
+  </div>
+</div>
```

Comment:
What's going on here? It doesn't seem right for this file to be in the root of the project.